organization/_common/s3.hcl

```
@@ -0,0 +1,21 @@
+locals {
+  account_vars = read_terragrunt_config(find_in_parent_folders("account.hcl"))
+}
+
+terraform {
+  source = "git::https://github.com/terraform-aws-modules/terraform-aws-s3-bucket?ref=v3.6.0"
+}
+
+inputs = {
+  acl = "private"
+
+  versioning = {
+    enabled = true
+  }
+
```

to make them really private
```suggestion
  block_public_acls                = true
  block_public_policy              = true
  ignore_public_acls               = true
  restrict_public_buckets          = true
  attach_require_latest_tls_policy = true
```